avm/res/cdn/profile/afdEndpoint/main.bicep

```diff
@@ -0,0 +1,81 @@
+metadata name = 'CDN Profiles AFD Endpoints'
+metadata description = 'This module deploys a CDN Profile AFD Endpoint.'
+metadata owner = 'Azure/module-maintainers'
+
+@description('Required. The name of the AFD Endpoint.')
+param name string
+
+@description('Conditional. The name of the parent CDN profile. Required if the template is used in a standalone deployment.')
+param profileName string
+
+@description('Optional. The location of the AFD Endpoint.')
+param location string = resourceGroup().location
+
+@description('Optional. The tags of the AFD Endpoint.')
+param tags object?
+
+@description('Optional. Indicates the endpoint name reuse scope. The default value is TenantReuse.')
+@allowed([
+  'NoReuse'
+  'ResourceGroupReuse'
+  'SubscriptionReuse'
+  'TenantReuse'
+])
+param autoGeneratedDomainNameLabelScope string = 'TenantReuse'
+
+@description('Optional. Indicates whether the AFD Endpoint is enabled. The default value is Enabled.')
+@allowed([
+  'Enabled'
+  'Disabled'
+])
+param enabledState string = 'Enabled'
+
+@description('Optional. The list of routes for this AFD Endpoint.')
+param routes array?
+
+resource profile 'Microsoft.Cdn/profiles@2023-05-01' existing = {
+  name: profileName
+}
+
+resource afd_endpoint 'Microsoft.Cdn/profiles/afdEndpoints@2023-05-01' = {
```

Comment:
Would suggest to use `afdEndpoint` as it is more consistent with the rest of the naming

Reply:
yep: https://github.com/Azure/bicep-registry-modules/pull/1106/commits/264d035f36513fe6c268ee0c65fc01340ca5eacd